app/src/pages/Quebec.tsx

```diff
@@ -0,0 +1,20 @@
+import { useTranslation } from "react-i18next";
+import { Box, Container, Typography } from "@mui/material";
+import { Link } from "../components/Link";
+import { LINKS } from "./links";
+export const Quebec = () => {
+  const { t } = useTranslation();
+  window.location.href = LINKS.DISCORD_QC;
```

Comment:
I think `window.location.replace` makes more sense for a redirect: https://developer.mozilla.org/en-US/docs/Web/API/Location/replace

Additionally, this should be in a `useEffect` hook so we don't have side effects running in the main render function.